data-raw/simulating-DAG.R

```diff
@@ -1,5 +1,62 @@
-# A DAG
 library(dagitty)
+<<<<<<< HEAD
+library(dplyr)
+library(survival)
+################################################
```

Comment:
You don't need to use these separators. Since we both use RStudio, its better to use RStudio's default separators ("Insert section separater" or something like that... Ctrl-Shift-R).